score.py

```diff
@@ -229,8 +229,9 @@ def convert_nifti_to_zipped_dicom_rt(nifti_file: Path, reference_series: Path, s
         modelId=model_id
     )
     # Log stdout, stderr from DICOM-RT conversion.
-    logging.debug("stdout: %s", stdout)
-    logging.debug("stderr: %s", stderr)
+    logging.info("DICOM-RT conversion stdout: %s", stdout)
```

Comment:
how noisy is that actually?

Reply:
Not at all, rtconvert only prints a single line on a successful conversion!